assets/js/index.js

```diff
@@ -154,6 +154,16 @@ $(document).ready(function () {
     }
   });
 
+  $("body").click(function(event) {
+    let sidebar = $(".sidebar");
```

Comment:
add an if condition that checks:
1. window width is less than 768px
2. sidebar has class `animate__slideInLeft`

then put the rest of the code of the function inside it.